package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "cli-table": "^0.3.1",
+    "@types/mkdirp": "^0.5.2",
```

Comment:
remove

Reply:
Actually not sure how this is working because we dont have `mkdirp` as a dependency